src/opm/output/eclipse/LoadRestart.cpp

```diff
@@ -58,6 +58,7 @@
 #include <functional>
 #include <initializer_list>
 #include <map>
+#include <unordered_map>
```

Comment:
We already include this header a few lines down.  There's no need to include it twice.

Reply:
Fixed